src/components/regularsite/regularsite.scss

```
@@ -0,0 +1,149 @@
+        &::after {
+            right: 10%;
+        }
+
+        .eye-pupils {
+            &::before,
+            &::after {
+                content: "●";
+                font-size: 27px !important;
+                position: absolute;
+                top: -21px;
+                font-size: 20px;
```

Stylelint (17.11.0)

[error] Duplicate property "font-size" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)